README.md

```diff
@@ -135,22 +140,15 @@ get '/profile' do
 end
 ```
 
-The API's response to our `POST` request will include our `access_token` and be returned into our `response` variable. Now we just need to parse out the `access_token` using the [json](http://flori.github.io/json/) gem.
+The API's response to our `POST` request will include the access token in a field called `access_token` in the `response` variable. Now we just need to parse out the access token.
 
 ``` ruby
 get '/profile' do
-    # Retrieve temporary Authorization Grant Code
-    session_code = request.env['rack.request.query_hash']['code']
-    
-    # POST Auth Grant Code + CLIENT_ID/SECRECT in exchange for our access_token
-    response = RestClient.post('https://github.com/login/oauth/access_token',
-                    # POST payload
-                    {:client_id => CLIENT_ID,
-                    :client_secret => CLIENT_SECRET,
-                    :code => session_code },
-                    # Request header for JSON response
-                    :accept => :json)
+
+    ...
+    [DEGW: I am adding ellipses here to highlight that the lines below are the ones we are interested in, but I don't know if this is just as confusing…]
```

Comment:
This section with the code snippets has definitely been one the main stumbling points so far. @hiimbex mentioned this section got too dense, which I agree with. I was trying to annotate each block of code as it was added. Perhaps a better way about this would be to post the entire code block with lots of comments?